cluster/src/main/java/org/apache/iotdb/cluster/utils/ClusterUtils.java

```diff
@@ -48,11 +50,24 @@
 
   public static final long START_UP_CHECK_TIME_INTERVAL = 5; // second
 
+  /**
+   * the data group member's heartbeat offset relative to the {@link ClusterConfig#getLocalDataPort()},
+   * which means the dataHeartbeatPort = getLocalDataPort() + DATA_HEARTBEAT_OFFSET.
+   */
+  public static final int DATA_HEARTBEAT_OFFSET = 1;
```

Comment:
How about using "DATA_HEARTBEAT_PORT_OFFSET" so it would be more clear what the port is for?

Reply:
sure, i will change it